transport/http/handler.go

```diff
@@ -50,6 +51,11 @@ type handler struct {
 func (h handler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	start := time.Now()
 
+	if req.URL != nil && strings.HasPrefix(req.URL.Path, "/debug/") {
```

Comment:
Discussed offline. For a variety of reasons (auth, rate limiting, middleware support), I think we ought to serve these pages on a separate port.

In the short term, let's have the open-source YARPC project register endpoints on `http.DefaultServeMux`. Our internal opinionated-config wrapper can call `http.ListenAndServe` on whichever port is appropriate.

In the long term, we should either build or use a more generic admin library. That library should handle registering and serving admin pages.